src/main/java/seedu/address/storage/PatientRecordsStorage.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.storage;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import seedu.address.commons.exceptions.DataConversionException;
+import seedu.address.model.PatientRecords;
+import seedu.address.model.ReadOnlyPatientRecords;
+
+/**
+ * Represents a storage for {@link PatientRecords}.
+ */
+public interface PatientRecordsStorage {
+
+    /**
+     * Returns the file path of the data file.
+     */
+    Path getPatientRecordsFilePath();
+
+    /**
+     * Returns CovigentApp data as a {@link ReadOnlyPatientRecords}.
+     *   Returns {@code Optional.empty()} if storage file is not found.
+     * @throws DataConversionException if the data in storage is not in the expected format.
```

Comment:
should there be an empty line before this? also, there are some extra white spaces in line 23

Reply:
I will change all the empty line thing. Thanks for pointing out!